### PR TITLE
More tweaks to compare-bedpes

### PR DESCRIPTION
**Pull request by apregier**

### CODE REVIEW

File: lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Genome::Model::Tools::Bedpe::EvaluateBedpes {
         },
         config_file => {
             is => 'Path',
-            doc => 'A tsv file which contains the following columns: caller_name, gold_name, data_name, bedpe, gold_bedpe, slop',
+            doc => 'A tsv file which contains the following columns: bedpe, gold_bedpe, slop (along with any other columns of metadata you wish to include',
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm]: This doc line needs a `)`!

File: lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm [hunk 2]
```diff
@@ -36,14 +36,12 @@ sub execute {
...
-            caller_name => $line->{caller_name},
-            gold_name => $line->{gold_name},
-            data_name => $line->{data_name},
-            stats => $stats,
-            slop => $line->{slop},
-        };
-        push @output, $entry;
+        for my $key (keys %$stats) {
+            $line->{$key} = $stats->{$key};
+        }
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm]: This loop could be replaced by `@$line{keys %$stats} = values %$stats;`.

**Comment by ebelter** [comment]
+1

File: lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm (hunk 2 above)
Comment (iferguson90) [inline lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm]: Is there a risk of `$stats` having a key that already exists in `$line`? For example, would this be necessary?
```perl
for my $key (keys %$stats) {
    die "Duplicate key $key: (descriptive message here)" if exists $line->{$key};
    $line->{$key} = $stats->{$key};
}
```

**Comment by iferguson90** [comment]
See comment, otherwise +1! :ear_of_rice:

**Comment by apregier** [comment]
Thanks for the comments, I made the minor changes and will merge after tests pass.

**Pull request merged**